dask_expr/io/tests/test_io.py

```diff
@@ -505,3 +507,11 @@ def test_from_dask_array():
     df = from_dask_array(arr, columns=["a", "b", "c", "d"])
     pdf = lib.DataFrame(arr.compute(), columns=["a", "b", "c", "d"])
     assert_eq(df, pdf)
+
+
+def test_from_dict():
```

Comment:
Should we test the error case here?

Reply:
They are covered in dask/dask, so I wasn't too careful here.